### PR TITLE
Added Vector Calculator Files

### LINKED ISSUE #192: Vector Calculator
## 🚀 Feature

A GUI based app built using Python that can fetch relation between 2 vectors along with the properties of each vector alone.

By relations I mean:
* Dot product.
* Cross product.
* Angle between both vectors.
* Component of each vector on the other.
* Projection of each vector on the other

By properties I mean:
* Unit Direction of each vector.
* Cosine Direction of each vector.
* Cosine Angles of each vector.
* Absolute Value of each vector. |A|

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

- [x] Yes

## Motivation

The project helps in explaining how Tkinter works, and how mathematical functions can be implemented in a program.

## Pitch

It's an on the go calculator to calculate multiple relations between vectors and making sure of on-paper calculations.

Issue made under GSSoC'22 contributions.

### PR DESCRIPTION
**Pull request by XZANATOL**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

- [x] Yes

### Description

Added Vector Calculator files.

Fixes #192 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #192 made under GSSoC'22 contributions

### CODE REVIEW

**Review by Kushal997-das — APPROVED** [review]
@XZANATOL great 

LGTM 👍 

over to you @photon149

File: Desktop Application/Basic/Python/Vector Calculator/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Cosine Angles of each vector.
+* Absolute Value of each vector. |A|
+
+## Required modules
+
+* Tkinter
+
+## Screenshots
+
+![image](./documents/Capture.PNG)
```
Comment (prakhar3062) [inline Desktop Application/Basic/Python/Vector Calculator/readme.md:26]: Image not visible  else is okay

Comment (XZANATOL) [reply Desktop Application/Basic/Python/Vector Calculator/readme.md:26]: I can't figure out the issue here as the image is visible on my PC.

image

And on my GitHub's fork.

image

Any ideas?

Comment (prakhar3062) [reply Desktop Application/Basic/Python/Vector Calculator/readme.md:26]: okay

**Review by prakhar3062 — COMMENTED** [review]
@photon149 Please review it. Okay from my side

**Review by photon149 — APPROVED** [review]
Looks good to me.
In case of no conflicts , merge the same @Kushal997-das

**Pull request merged**